README.md

```diff
@@ -296,6 +301,11 @@ max_request_size = 1048576  # 1MB
 - **delivery**: Handles transaction submission to multiple chains (supports per-network account mapping)
 - **discovery**: Sources for discovering new intents (on-chain events, off-chain APIs)
 - **order**: Execution strategy and protocol-specific settings
+  - `order.execution_strategy.config` now accepts optional pricing fields:
+    - `pricing_currency` (default: "USDC")
+    - `commission_bps` (default: 20)
+    - `gas_buffer_bps` (default: 1000)
+    - `rate_buffer_bps` (default: 14)
```

Comment:
Nit; This seems to be high-level overview. Probably belongs in different documentation (technical/config) we can add later.

```suggestion
```